documentation/docs/authentication.md

```diff
@@ -180,3 +186,40 @@ Use the following options to configure the KMS server for Microsoft Entra Id:
 --jwks-uri=https://login.microsoftonline.com/<TENANT_ID>/discovery/v2.0/keys
 --jwt-audience=<CLIENT_ID>
 ```
+
+## Authenticating using an API Token
+
+The server can be configured to authenticate using an API token passed in the `Authorization` header.
+
+To proceed, follow these steps:
+
+- run Cosmian KMS server without API token authentication
+- generate a symmetric key and export it from the server
+- restart the server with the `--api-token-id` option
+- configure `ckms` client with a `kms_access_token` containing the API token in base64.
```

Comment:
again - check what happen when you use `ckms login`

Reply:
Ok - do you think that's ok? It means we keep the `login` command only for the jwt_auth with IDP, needing oauth2 config.
Maybe add a mention in the login command CLI's details - that this should be used only if the user is using an IDP to login ?

I thought about displaying info of api_token auth insteas of this error - but this might be confusing in the end to handle both in this command.

Reply:
changing the error message to:
```
            CliError::Default(format!(
                "The `login` command (only used for JWT authentication) requires an Identity \
                 Provider (IdP) that MUST be configured in the oauth2_conf object in {conf_path:?}"
            ))
``` 

done in cfb2b3dfe03fe68ab1399b42e8efe0ec5420cc9d